src/cluster-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mz-cluster-client"
-description = "The public API for both compute and storage"
+description = "Types for storage and compute."
```

Comment:
Why's this changing? `cluster-client` isn't just for types, is it?

Reply:
ill clean this up